src/assets/wise5/components/aiChat/ai-chat-student/ai-chat-student.component.spec.ts

```diff
@@ -45,7 +54,12 @@ describe('AiChatStudentComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  it('should not show response from a connected component', async () => {
+    component.processConnectedComponentState({ studentData: { response: 'test response' } });
+    const loader = TestbedHarnessEnvironment.loader(fixture);
+    await (await loader.getHarness(MatButtonHarness)).click();
+    const messages = fixture.debugElement.queryAll(By.css('a-chat-student-message'));
```

Comment:
There is a typo here. The selector should be `ai-chat-student-message`. As discussed, this test also will return true no matter what selector is used. Should probably be modified to look for a length of 1 or something to make the test more productive.